pkg/auth/routes.go

```diff
@@ -168,10 +168,15 @@ func NewRouter(p *deps.RootProvider, configSource *configsource.ConfigSource) *h
 	)
 	webappAuthEntrypointChain := httproute.Chain(
 		webappPageChain,
+		p.Middleware((newRequireAuthenticationEnabledMiddleware)),
```

Comment:
Do not add the middleware to webappAuthEntrypointChain. Add a new chain for /login and /signup, and add the middleware there.

Reply:
Updated